Client/mods/deathmatch/logic/CFileReader.cpp

```diff
@@ -59,3 +59,19 @@ bool CFileReader::LoadFileToMemory(const SString& strFilePath)
     }
     return false;
 }
+
+bool CFileReader::LoadDataBufferToMemory(const SString& buffer)
+{
+    std::streamsize m_BufferSize = buffer.size();
+    if (m_BufferSize == eIFSTREAM::SIZE_ERROR)
```

Comment:
can you please rename `m_BufferSize` to `iBufferSize` for consistency? I made the same mistake within `CFileReader::LoadFileToMemory`, so please do me one more favor and rename `m_iFileSize` to `iFileSize`.

Reply:
Sure. Done in 0cf25a0.